script.js

```diff
@@ -31,51 +36,44 @@ function addEvents() {
     });
   });
 }
+
 // populate dom with the list
 const addBtn = document.querySelector('#add-btn');
 function showBooks(list) {
-  const booksListDiv = document.querySelector('#books-container');
-  list.forEach((lists, i) => {
+  const booksListDiv = document.querySelector('#List');
+  // let colorChange = 0;
```

Comment:
- [ ] Kindly please, remove this introduced old piece of commented-out code.  That way, the project will be cleaner 💪